engine/table/src/main/java/io/deephaven/engine/table/impl/naturaljoin/IncrementalNaturalJoinStateManagerTypedBase.java

```diff
@@ -176,6 +184,7 @@ protected void buildTable(
                     migrateFront(modifiedSlotTracker);
                 }
 
+                bc.resetSharedContexts();
```

Comment:
I must have been hallucinating.  Probably I was using "ii" originally instead of "k", which was NFG.  There are no shared contexts involved in a single column naturalJoin, so the shared context is null to begin with.